onnxscript/rewriter/onnx_fusion_rule_sets_test.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import unittest
+from typing import Any
+
+import numpy as np
+import onnx
+import onnx.reference
+
+from onnxscript import ir
+from onnxscript.rewriter import onnx_fusion_rule_sets
+
+FLOAT = onnx.TensorProto.FLOAT
+FLOAT16 = onnx.TensorProto.FLOAT16
+INT64 = onnx.TensorProto.INT64
+
+
+class OnnxFusionRuleTest(unittest.TestCase):
+    def _get_random_inputs(self, model: onnx.ModelProto) -> dict[str, Any]:
+        feeds: dict[str, Any] = {}
+        for i in model.graph.input:
+            ish = tuple(i.type.tensor_type.shape.dim)
+            # Creates an input tensor with a dimension defined by the onnx model
+            # or equals to i + 2 with i being the dimension index.
+            # The tensor is kept small to make the test fast.
+            shape = tuple(
+                (d.dim_value if d.dim_value > 0 else i + 2) for i, d in enumerate(ish)
+            )
+            if i.type.tensor_type.elem_type == onnx.TensorProto.FLOAT:
+                feeds[i.name] = np.random.randn(*shape).astype(np.float32)
+            elif i.type.tensor_type.elem_type == onnx.TensorProto.FLOAT16:
+                feeds[i.name] = np.random.randn(*shape).astype(np.float16)
+            elif i.type.tensor_type.elem_type == onnx.TensorProto.INT64:
+                feeds[i.name] = np.arange(0, shape[0]).astype(np.int64)
+            else:
+                raise AssertionError(f"Not implemented for input {i}")
+        return feeds
+
+    def _check_model(
+        self,
+        model: onnx.ModelProto,
+        optimized_model: onnx.ModelProto,
+        feeds: dict[str, Any] | None = None,
+        atol: float = 0.0,
+        rtol: float = 1e-7,
+        use_ort: bool = False,
+    ):
+        # import onnxruntime does not because of the subfolder named onnxruntime
+        import os
+        import sys
+
+        sys_path = sys.path
+        sys.path = [os.path.join(onnx.__file__, "..", "..")] + [
+            p for p in sys_path if "onnxruntime" in p
+        ]
+        import onnxruntime
```

Comment:
hmm

Reply:
The right fix is to rename onnxruntime subfolder.

Reply:
See #1786.